CHANGELOG.md

```diff
@@ -1,4 +1,17 @@
 # Changelog
+### 2.23.2
+This release includes:
+* An Amazon Linux 2 Base
+* Fluent Bit [1.8.14](https://fluentbit.io/announcements/v1.8.14/)
+* Amazon CloudWatch Logs for Fluent Bit 1.7.0
+* Amazon Kinesis Streams for Fluent Bit 1.9.0
+* Amazon Kinesis Firehose for Fluent Bit 1.6.1
+
+Compared to `2.23.1` this release adds:
+* Enhancement - Mitigate throttling issue on log group in cloudwatch_logs plugin [fluentbit:4826](https://github.com/fluent/fluent-bit/pull/4826)
```

Comment:
What about the bug fixes?
https://github.com/fluent/fluent-bit/commit/fd9b0cde8e9f66bfbaad0c16012670124e919ad6 aws: check AWS EC2 IMDSv2 client error

https://github.com/fluent/fluent-bit/commit/360670a4d953cd570c32b2391c800eec8f8c8fc3 out_cloudwatch_logs: fix build issue from https://github.com/fluent/fluent-bit/pull/4826

Reply:
I think we should add this one as well: out_cloudwatch_logs: fix build issue.

Reply:
1. check AWS EC2 IMDSv2 client error is a small fix. 
2. out_cloudwatch_logs: fix build issue from is because of the CR for throttling mitigation which is not an existing ERROR in 2.23.1. 

From my perspective, The main goal to mention inside AWS for Fluent Bit release is to announce the AWS related and customer facing feature and issue . It's not required to list every small bug and features. For details customer could check the Fluent Bit announcement  which is linked inside release note.